pkg/autoops/api/api.go

```diff
@@ -825,19 +889,50 @@ func (s *AutoOpsService) isNoUpdateAutoOpsRuleCommand(req *autoopsproto.UpdateAu
 
 func (s *AutoOpsService) createUpdateAutoOpsRuleCommands(req *autoopsproto.UpdateAutoOpsRuleRequest) []command.Command {
 	commands := make([]command.Command, 0)
+	var opsType *autoopsproto.OpsType = nil
 	if req.ChangeAutoOpsRuleOpsTypeCommand != nil {
 		commands = append(commands, req.ChangeAutoOpsRuleOpsTypeCommand)
+		opsType = &req.ChangeAutoOpsRuleOpsTypeCommand.OpsType
 	}
+
 	for _, c := range req.AddOpsEventRateClauseCommands {
+		if opsType != nil {
+			if *opsType == autoopsproto.OpsType_DISABLE_FEATURE {
+				c.OpsEventRateClause.ActionType = autoopsproto.ActionType_DISABLE
+			} else if *opsType == autoopsproto.OpsType_ENABLE_FEATURE {
+				c.OpsEventRateClause.ActionType = autoopsproto.ActionType_ENABLE
+			}
+		}
```

Comment:
I'm not sure what this part is trying to do.
`ChangeAutoOpsRuleOpsTypeCommand` is deprecated but `req.ChangeAutoOpsRuleOpsTypeCommand.OpsType` is referenced.

Reply:
To support multi-scheduling, Enable and Disable have been changed from being managed by OpsType to being managed by ActionType held within Clause.

As mentioned in this comment, if you change the management screen, this code will no longer be needed and will be deleted.